### PR TITLE
Encoding mgw labels to compatible with control plane.

### PR DESCRIPTION
**Pull request by dinusha92**
### Purpose

APIM endpoint has been changed to accept the environment in the base64 encoded format. This PR makes adapter compatible with that change.
This PR contains improvments related to the implementation in https://github.com/wso2/product-microgateway/issues/1594
### Issues

Fixes https://github.com/wso2/product-microgateway/issues/1594

### Automation tests
 - Unit tests added: No
 - Integration tests added: No

### Tested environments

Not Tested

---
#### Maintainers: Check before merge
- [x] Assigned 'Type' label
- [x] Assigned the project
- [x] Validated respective github issues
- [x] Assigned milestone to the github issue(s)

### CODE REVIEW

**Comment by praminda** [comment]
Refer the PR to original `fetchAPI` impl related git issue and theres conflict

File: resources/conf/config.toml [hunk 1]
```diff
@@ -86,7 +86,7 @@ enabled = false
 serviceUrl = "https://localhost:9443/"
 username="admin"
 password="admin"
-environmentLabels = ["Production and Sandbox"]
+environmentLabels = ["default"]
```
Comment (praminda) [inline resources/conf/config.toml:89]: Here the default should be `"Production and Sandbox"` after viraj's latest changes I believe
```suggestion
environmentLabels = ["Production and Sandbox"]
```

Comment (dinusha92) [reply resources/conf/config.toml:89]: There was a discussion. hence I changed this. I'll revert this back then. I wasn't aware of Viraj's feature.

Comment (dinusha92) [reply resources/conf/config.toml]: This commit is reverted.

**Review by praminda — APPROVED** [review]

**Pull request merged**